pandas/core/generic.py

```diff
@@ -546,13 +546,16 @@ def swaplevel(self, i, j, axis=0):
     _shared_docs['rename'] = """
         Alter axes input function or functions. Function / dict values must be
         unique (1-to-1). Labels not contained in a dict / Series will be left
-        as-is.
```

Comment:
these are shared between Series & DataFrame yes? if so, then need to do something to only show the ``.rename(scalar)`` for ``Series``